yb-voyager/cmd/importData_test.go

```diff
@@ -846,6 +853,636 @@ CREATE TABLE public.foo (
 	4. Import Data File tests
 */
 
+////=========================================
+
+// HELPER functions for live
+func isExportDataStreamingStarted(t *testing.T) bool {
+	//check for status until its streaming
+	statusFilePath := filepath.Join(exportDir, "data", "export_status.json")
+	status, err := dbzm.ReadExportStatus(statusFilePath)
+	if err != nil {
+		utils.ErrExit("Failed to read export status file: %s: %v", statusFilePath, err)
+	}
+	testutils.FatalIfError(t, err, "reading dbzm export status")
+	if status != nil && status.Mode == dbzm.MODE_STREAMING {
+		return true
+	}
+	return false
+}
+
+func snapshotPhaseCompleted(t *testing.T, postgresPass string, targetPass string, snapshotRows int64, tableName string) bool {
```

Comment:
nit: for func names that are helper assertions, I'd recommend the pattern `assert*` . This sounds more like `AssertSnapshotPhaseCompleted`

Reply:
These functions are no assertions here; it is just waiting for a particular phase to complete before proceeding further. e.g., asserting whether snapshot rows are exported and imported, and then proceeding to the streaming phase, and so on.